app/tasks/sdks.php

```diff
@@ -239,8 +239,8 @@
                         cd ' . $target . ' && \
                         git init --initial-branch=' . $gitBranch . ' && \
                         git remote add origin ' . $gitUrl . ' && \
-                        git fetch && \
-                        git pull ' . $gitUrl . ' && \
+                        git fetch origin ' . $gitBranch . ' && \
+                        git pull origin ' . $gitBranch . ' && \
```

Comment:
Would you please update the description of the PR with some more info on some of these git changes?